### ISSUE TITLE
pip support

### ISSUE THREAD

**Issue by kunguz** [mritools/cupyimg#5]
Are you planning to add `cupyimg` to pip anytime soon so that people can add this library to their `requirements.txt` in their projects?

**Comment by grlee77** [comment]
It would be easy enough to put a source release on PyPI, but I am a bit reluctant to do it as the current plan is to move this repository under NVIDIA's [RAPIDS](https://rapids.ai) organization instead. That way it can benefit from automated testing on GPU hardware, etc. prior to making an official release. See cupy/cupy#3531 for the initial discussion related to this. I have had some correspondence via email with NVIDIA regarding the logistics of the move, but do not yet know when the new repository will become available. I intend keep this repository here as well, but will probably mark it as ARCHIVED once the migration to RAPIDS is complete.

The new RAPIDS `cuImage` module will probably contain only the things currently under the `cupyimg.skimage` module. I am in the process of porting the items from `cupyimg.scipy` and `cupyimg.numpy` upstream to CuPy itself.

Out of curiosity, which functions are you primarily interested in? If there are still functions missing that would be important for your application, please let me know as that may aid in deciding which features to implement going forward.

**Comment by kunguz** [comment]
Thanks for providing a detailed reply! I am interested in using `gradient`, `polyfit`, and `poly1d` functions, which aren't readily available in Cupy.

**Comment by grlee77** [comment]
`polyfit` was recently merged into CuPy master: cupy/cupy#3747 and there is an open PR containing `poly1d` at cupy/cupy#3547

IIRC, a contributor had offered to work on `gradient` over there as well, but I don't think it ever materialized. I will try to port the `gradient` function from here when I get a chance. It is pretty easy to copy over the function, but it takes some time to expand the tests and get them to match CuPy's style.
